libs/dal/src/repositories/job/job.schema.ts

```diff
@@ -101,10 +101,13 @@ const jobSchema = new Schema<JobDBModel>(
       type: Schema.Types.ObjectId,
       ref: 'Subscriber',
     },
+    expireAt: Schema.Types.Date,
   },
   schemaOptions
 );
 
+jobSchema.index({ expireAt: 1 }, { expires: '48h' });
```

Comment:
Maybe we can extract the 48H to a reusable variable so we can update all instances easily?